### DISCUSSION TITLE
是否有办法在批量复制时跳过已存在的文件

### DISCUSSION THREAD

**Discussion by ifloppy** [AlistGo/alist#6002] [Q&A]
如题。想请教一下各位大佬。
如果我在Alist上把一个文件夹从一个地方复制到另一个地方，中途重启Alist，任务队列应该会丢失，此时我能否在Alist上在执行相同的复制操作，但是跳过已复制的文件？

**Comment by pongfcnkl** [comment]
用这个项目
https://github.com/lee-cq/alist-sync
